src/YieldDisburser.sol

```diff
@@ -123,6 +123,26 @@ contract YieldDisburser is OwnableUpgradeable {
         lastClaimedBlockNumber = _lastClaimedBlockNumber;
     }
 
+    /**
+     * @notice Returns the current distribution of voting power for projects
+     * @return address[] The current breadchain projects
+     * @return uint256[] The current distribution of voting power for projects
+     */
+    function getCurrentVotingDistribution() public view returns (address[] memory, uint256[] memory) {
+        return (projects, projectDistributions);
+    }
+
+    /**
+     * @notice Return the current voting power of a user
+     * @param _account Address of the user to return the voting power for
+     * @return uint256 The voting power of the user
+     */
+    function getCurrentVotingPower(address _account) public view returns (uint256) {
+        uint256 votingPower =
+            this.getVotingPowerForPeriod(lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account);
+        return (votingPower < minRequiredVotingPower ? 0 : votingPower);
```

Comment:
```suggestion
        return (votingPower < minRequiredVotingPower ? maxPoints : votingPower);
```

Reply:
don't understand this suggestion? why would we return maxPoints?